honeybee_radiance/lib/modifiers.py

```diff
@@ -1,17 +1,39 @@
 """Library of default modifiers for honeybee-radiance.
 
-Default values are generic values to set the initial visible reflectance and
-transmittance values in your model. There is no guarantee that these values exactly matches your model.
+Default values are generic values to set the initial visible / solar reflectance and
+transmittance values in your model. There is no guarantee that these values exactly
+matches your model.
```

Comment:
"match what you are trying to model?"
I am pretty sure that the match the model this is built/exported.